docs/template-vm.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: linux-vm-template
+  annotations:
+    description: "OCP kubevirt linux, template"
+    tags: "kubevirt,ocp,template,linux"
+objects:
+- apiVersion: kubevirt.io/v1alpha1
+  kind: OfflineVirtualMachine
+  metadata:
+    name: linux-vm-${NAME}
+  spec:
+    template:
+      metadata:
+      labels:
+      spec:
+        domain:
+          resources:
+            memory: ${MEMORY}
+            cpu: ${CPU}
+          os:
+            machine: pc-q35-rhel7.4.0
+          devices:
+            interface:
+              model:
+                type: virtio
+            disks:
+              - name: disk0
+                volumeName: disk0-pvc
+                disk:
+                  dev: vda
+                  bus: virtio
+  volumes:
+    - name: disk0-pvc
+      persistentVolumeClaim:
+        claimName: linux-vm-pvc-${NAME}
+- apiVersion: kubevirt.io/v1alpha1
+  kind: PersistentVolumeClaim
+  metadata:
+    name: linux-vm-pvc-${NAME}
+labels:
```

Comment:
this label would be on the template metadat further above (where I commented that it is missing)

Reply:
Ok, I placed in the wrong spot, will move above

Reply:
@fabiand @fromanirh   
The labels are at the bottom of the file on purpose, and in the form of:

labels:  
      **template**: “miq.github.io/kubvirt/vm-template”  

“miq.github.io/kubvirt/vm-template”  label applied to all objects created from this template 
and without having a flag of true/flase

the syntax is mentioned here:
https://docs.openshift.org/latest/dev_guide/templates.html#writing-labels

I'm open to other suggestions/corrections here